utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -249,7 +251,7 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
 #endif
             std::vector<float> area = {0.08, 1};
             std::vector<float> aspect_ratio = {3.0f/4, 4.0f/3};
-            input1 = rocalJpegCOCOFileSourcePartial(handle, path, json_path.c_str(), color_format, num_threads, false, area, aspect_ratio, 10, false, false, ROCAL_USE_USER_GIVEN_SIZE_RESTRICTED, decode_max_width, decode_max_height);
+            input = rocalJpegCOCOFileSourcePartial(handle, path, json_path.c_str(), color_format, num_threads, false, area, aspect_ratio, 10, false, false, ROCAL_USE_USER_GIVEN_SIZE_RESTRICTED, decode_max_width, decode_max_height);
```

Comment:
color_format is not suirable with tensor. Please use layout instead

Reply:
The color format needs to be passed to the decoder which specifies whether it is RGB, BGR or Grayscale formats.